lib/components/related_artists/index.js

```diff
@@ -16,24 +17,30 @@ class RelatedArtists extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = this.layoutState()
+    this.state = {
+        columns: 0,
+        imageSize: {
+          width: 1,
```

Comment:
Happy to hear suggestions though because this does feel a bit inelegant. I'm guessing this could be fixed by doing some work in the image view or by not rendering images at all until after layout?